server/pages/index.js

```diff
@@ -127,7 +127,8 @@ function getDefaultContext( request ) {
 		clientIp: request.ip ? request.ip.replace( '::ffff:', '' ) : request.ip,
 		isDebug: context.env === 'development' || context.isDebug,
 		tinymceWpSkin: context.urls[ 'tinymce/skins/wordpress/wp-content.css' ],
-		tinymceEditorCss: context.urls[ 'editor.css' ]
+		tinymceEditorCss: context.urls[ 'editor.css' ],
+		commitChecksum: getCurrentCommitShortChecksum()
```

Comment:
Smart! I was even thinking about webpack, but even moving it outside of the handlars is enough. Thanks!